about.html

```diff
@@ -0,0 +1,109 @@
+            participate in activities to spread the value of sharing, free culture, and open knowledge with CC .</p>
+          <p>Free software supporters, Wikipedia activists, open knowledge foundation activists, art galleries,
+            libraries, museums, archives, governments,
+            foundations, lawyers, social activists, etc. various people active in the field of open content and open
+            knowledge gather in one place to
+            share richer and more dynamic We hope that it will be an opportunity to share our thoughts on creating an
+            ecosystem.</p>
+          <p class="diamonds">◆◆◆</p>
+          <p class="lorem-ipsum">Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis repellat iure
+            laboriosam quae, voluptatem distinctio ducimus mollitia sed voluptate sit aliquid dicta voluptates, minima
+            explicabo ea? Culpa dolores eos vel voluptas, in inventore autem numquam aperiam excepturi, animi corrupti
+            nam error molestias minima id facilis suscipit deleniti, quibusdam quae. Reiciendis optio fugiat libero quo
+            at nisi eaque impedit. Laboriosam nulla quod sed blanditiis commodi aliquid repellat, excepturi voluptates
+            eveniet expedita porro illum dolor, consequatur laborum possimus iste, quae perferendis eius?</p>
+        </div>
+        <div class="questions">Do you have any other questions about Orange Studio's Launch Party?<span class="email">
+            melaniesigrid@protonmail.com</span></div>
+      </div>
+    </section>
+    <section class="logo-contest">
+      <hr class="gray-separator">
+      <h3 class="logo-about">Orange Studio's Logo</h3>
+      <hr class="underline">
+      <div class="logo-image"><img src="images/Orange.png" alt="Orange Studio Logo" class="logo-section"></div>
+      <p class="logo-description">Our logo was designed by Melanie Sigrid. "Orange is a colour that represents energy
+        and youth. It shows our youthful energy, and fruits represent fresh ideas taking shape."</p>
+    </section>
+    <section class="future-events">
+      <hr class="gray-separator">
+      <h3 class="logo-about">Our Future Events Together</h3>
+      <hr class="underline">
+      <div class="future-img-container">
+        <div class="future"><h2 class="future-title">Design Trends 2k21</h2></div>
+        <div class="future"><h2 class="future-title">Creative Coding Workshop</h2></div>
+      </div>
+    </section>
+  </main>
+  <footer class="footer-about-section">
+    <br>
+    <br>
+    <div class="footer-container">
+      <img class="logo footer-logo" src="images/Orange.png" alt="Orange Studio Logo">
+      <p class="footer-text">© 2021 Melanie Sigrid. Some rights reserved. View full license <a
+        href="https://github.com/melaniesigrid/OrangeStudio/blob/main/MIT.md" target="_blank"
+        rel="noopener noreferrer">here</a>.</p>
+    </div>
+  </footer>
```

Comment:
- [ ] On desktop view the footer has a dark background and on mobile view, it has a white background. Please make this change on both pages.

Reply:
Thank you for the review! Upon another inspection of the design, the difference in the footer color is not a change between the desktop and mobile view. Dark and light colors alternate between sections, the dark footer is meant for the About section, and the light footer for the main page (Both the same on desktop & mobile). Otherwise, the colors would not alternate and it would not look good...